fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/processor/impl/CosExportImpl.java

```diff
@@ -161,13 +163,48 @@ public Parameters exportPatient(String logicalId, MediaType outputFormat, Instan
         }
     }
 
-    // not implemented yet
     @Override
```

Comment:
on line 123, can you remove the printstacktrace

Reply:
done